examples/custom-mods/custom_mods/task.py

```diff
@@ -0,0 +1,122 @@
+"""custom_mods: A Flower app with custom mods."""
+
+import warnings
+from collections import OrderedDict
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+from flwr_datasets import FederatedDataset
+from flwr_datasets.partitioner import IidPartitioner
+from torchvision.transforms import Compose, Normalize, ToTensor
```

Comment:
```suggestion
import torch
import torch.nn as nn
import torch.nn.functional as F
from flwr_datasets import FederatedDataset
from flwr_datasets.partitioner import IidPartitioner
from torch.utils.data import DataLoader
from torchvision.transforms import Compose, Normalize, ToTensor
```

Same as above for sorting imports.